server/sonar-web/src/main/js/components/icons-components/SettingsIcon.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+
+interface Props {
+  className?: string;
+  fill?: string;
+  size?: number;
+  style?: React.CSSProperties;
+}
+
+export default function SettingsIcon({
```

Comment:
It should be added to the `./icons.ts` files. I'm starting to think that maybe that file should be generated automatically during webpack job ?

Reply:
well, I'm not a big fan of exposing everything we have :) as soon as something is exposed, then it's hard to change it

Reply:
Ah yeah true... I didn't think about that... Ok lets expose as we need only!